### DISCUSSION TITLE
Reolink camera problem with 'Connection interrupted' and 'Reconnecting' not solved

### DISCUSSION THREAD

**Discussion by robtperk** [koush/scrypted#1005] [Q&A]
Two of my Reolink cameras (or perhaps both cams on the Duo are experiencing this problem. The other 2 Reolink cameras are fine. I can't even delete the camera(s) to re-add because that isn't an option.

**Comment by Gustovier** [comment]
I’m also having this problem. Has anybody resolved?

**Comment by Slabonski** [comment]
If you were running Scrypted on Home Assistant OS then this is a possible workaround. I had this exact same thing happen when adding a Duo 2 camera to either the ONVIF or Reolink plugins while running on HAOS. The problem (as I see it) is caused by the size of the snapshot from the Duo 2. There are two workarounds to this. The first is to change the Clear Stream resolution on the camera from 4608x1728 to 4096x1536. The other is to add a snapshot url which includes width and height parameters. The syntax is: http://(ip address)/cgi-bin/api.cgi?cmd=Snap&channel=0&rs=wuuPhkmUCeI9WG7C&user=admin&password=(password)&width=1536&height=576 make sure the camera is disconnected from the network so you can get back into the plugin to add the snapshot url. Both of these worked in my situation. BTW I determined this by messing with the snapshot url in the RTSP plugin. The RTSP plugin did not present this same problem until I used a snapshot url without width and height constraints.
